imfilebrowser.h

```diff
@@ -90,8 +90,24 @@ namespace ImGui
         // default value is 0 (the first type filter)
         void SetCurrentTypeFilterIndex(int index);
 
+        // when ImGuiFileBrowserFlags_EnterNewFilename is set
+        // this function will pre-fill the input dialog with a filename.
+        void SetInputName(std::basic_string<char> input) noexcept
+        {
+            if((flags_ & ImGuiFileBrowserFlags_EnterNewFilename)) {
```

Comment:
curly braces appear on new lines in other functions

Reply:
you could use a clangformat :) I'll fix.